README.md

```diff
@@ -1,28 +1,78 @@
 # github-action-detect-unused-secrets
 
-GitHub Action that automatically detects unused secrets comparing the workflow secret names with the repo secret names.
+GitHub Action that detects unused secrets.
+Compares the secret names in your repository with the secret names under `.github/workflows`
```

Comment:
let's clarify better what this does. it checks which secrets are used in workflows with those defined in the repository settings. let's also clarify very well that this action doesn't get access to the values of secrets, just their names